draft-ietf-quic-transport.md

```diff
@@ -2740,8 +2740,10 @@ which is comprised of:
   the initial salt defined in section 5.2 of {{QUIC-TLS}}.
 
 A server advertises these values using a NEW_TOKEN frame {{frame-new-token}}.
-The token MUST include or associated with the alternative version number with
-which it can be used.
+The token MUST permit the server to recover at least the alternative version
+number being associated to the token.  This property can be achieved for example
```

Comment:
It MUST be able to recover the salt as well, I think.  It's possible that version number maps directly to salt -- but then, that's one implementation of being able to recover it.

Reply:
Good point! Fixed in 757e5e0.